chef/cookbooks/ceph/recipes/osd.rb

```diff
@@ -69,7 +69,15 @@
   if is_crowbar?
     node.set["ceph"]["osd_devices"] = [] if node["ceph"]["osd_devices"].nil?
     min_size_blocks = node["ceph"]["osd"]["min_size_gb"] * 1024 * 1024 * 2
-    unclaimed_disks = BarclampLibrary::Barclamp::Inventory::Disk.unclaimed(node).sort.select {|d| d.size >= min_size_blocks}
+
+    claim_string = "Ceph"
```

Comment:
I would prefer snakecase claim strings as we anyway display some I18n key within crowbar.